service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java

```diff
@@ -365,6 +373,88 @@ void shouldNotHaveCaseIdPropertyWhenRoboticsDataDeserializes() throws IOExceptio
         }
     }
 
+    @Test
```

Comment:
Very minor but it might be worth re-ordering these so we get happyBoth `assertDoesNotThrow` tests first.